### ISSUE TITLE
Icon for new articles not yet displayed in article_box

### ISSUE THREAD

**Issue by QuietEvolver** [EwRicklefs/Watch-the-screens#35]
This will be updateded with a "Breaking News" type icon for a new story now yet displayed in the article_box and if the database.aritcles.displayed = false, then it will continue to appear; if the new story has been displayed in the article_box and if the database.aritcles.displayed = true, then the "breakingNews=icon' will cease to display.  
StickyFooterTicker: breaking news icon will be displayed for new articles until the article_box. flag from the server has been checked to a 'displayed'. 
ArticleBox: a new field will be added into the database table by the database manager; this will allow the the breaking news icon to be contingent on the value of said field.

**Comment by QuietEvolver** [comment]
Fleshed out Role for Admin - URL visibility #14

**Comment by QuietEvolver** [comment]
Link to the new article spec:
#46
